### DISCUSSION TITLE
Thoughts and questions

### DISCUSSION THREAD

**Discussion by sfbth** [bpatrik/pigallery2#349] [General]
Hello

Love the app, it's absolutely perfect for my needs, nothing unnecessary, slick and fast and easy to understand - great job chap. 

I do have a few thoughts and questions though and don't want to log loads of bugs/feature requests.

1) Unzipping - the zip file seems to contain the pigallery2 subdirectory so, when unzipping to -d pigallery you get two pigallery folders - the path to my install contains "/pigallery2/pigallery2/"

2) Viewing pictures - when you browse the albums and click on a picture it appears as low-res for a second or two then snaps into the proper resolution. I can't work out what's causing that, please could you assist?

3) I'm not a dev but was wondering if there was something I can contribute - perhaps I could make a start on the wiki by putting together the headings and structure and fill in what I can? 

4) The concept of the nested jobs with "run after", etc with the progress bars is really nice - good stuff!

5) I would prefer that non-admins do not see the message that there are issues that the admin need to sort, I think that's just extraneous and confusing for the user.

Cheers
Ben

**Comment by bpatrik** [comment]
Thanks, I'm happy that people like the app. Here are my thoughts: 

1. yeah that could be improved
2. that is a feature, not a bug :), while the bigger res photo loads, it shows the thumbnail and not a black screen
3. I was also thinking about putting together some userguide / FAQ thing as I get similar usage related questions  multiple times. On the other hand, I am laizy / do not have time to keep all code,tests,userguide always in sync/up-to-date.
4. Thanks :)
5. I do not mind making it switchable.

**Comment by sfbth** [comment]
Hello

1. I think you just need to tweak your instructions to remove the directory parameter, job done.

3. Well, let me know if there's any way I can help.

5. That sounds good.

Cheers chap

Ben

________________________________
